web/src/beta/lib/core/StoryPanel/Block/builtin/Title/index.tsx

```diff
@@ -26,14 +26,20 @@ const TitleBlock: React.FC<Props> = ({ block, isSelected, ...props }) => {
     [property?.title?.color?.value],
   );
   const hasEmptySpace = isEmptyString(title);
+  const titleProperty = useMemo(() => {
+    return {
+      ...property,
+      panel: { padding: property?.title?.padding },
+    };
+  }, [property]);
```

Comment:
Can you do this up a level?
You are already pulling out title from the page and then pass that title property in to this Title "story block". If you process fully where you pull the title group out of page we just have one place to look for issues in processing in the future or adding new fields/groups/etc